### DISCUSSION TITLE
Cant install pr-1282

### DISCUSSION THREAD

**Discussion by LucDeGit** [FlareSolverr/FlareSolverr#1299] [General]
Hello, 

I want to install the pr-1282 to my flaresolverr on my Truenas but when i replace the tag by "pr-1282" and try to deploy it that not working because he dont found it.
Any idea on to solve that ? 

Thank for your help ;)

**Comment by ilike2burnthing** [comment]
You're specifically using `ghcr.io/flaresolverr/flaresolverr:pr-1282`, not just `flaresolverr/flaresolverr:pr-1282`, which is DockerHub?

Are you using a Docker compose file or a Docker run command, or are you using Portainer's GUI or something similar?

**Reply by LucDeGit** [reply]
Hello, 

I use the "custom app" of Truenas scale that use Kubernetes and its look like  ![image](https://github.com/user-attachments/assets/110ca42b-7358-49a1-bb2c-fe95f087d3da) and when I put pr-1282 that not working :/

**Reply by LucDeGit** [reply]
Ok after some research i find how to push the update pr-1282 on truenas via the custom app 

For the people searching for help you need to put the container image like this : 
![image](https://github.com/user-attachments/assets/766b5eff-f4a4-49f2-aa2a-c390731477f1)

Thank @ilike2burnthing for your time to reply me <3
